galaxy_ng/app/api/ui/serializers/synclist.py

```diff
@@ -0,0 +1,106 @@
+import logging
+
+from rest_framework import serializers
+
+from pulp_ansible.app import models as pulp_ansible_models
+
+from galaxy_ng.app import models
+
+log = logging.getLogger(__name__)
+
+
+class SyncListCollectionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = pulp_ansible_models.Collection
+        fields = ["namespace", "name"]
+
+
+class SyncListCollectionSummarySerializer(serializers.Serializer):
+    namespace = serializers.CharField(max_length=64)
```

Comment:
Looks like these numbers may appear elsewhere. Probably better to use them as constants.